styles.css

```diff
@@ -370,6 +368,78 @@ h4 {
   background: #2a2b33;
 }
 
+#popup {
+  display: none;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  backdrop-filter: blur(15px);
+  -webkit-backdrop-filter: blur(15px);
+  background-color: rgba(0, 0, 0, 0.3);
+  overflow: auto;
+}
+
+#popup > * {
+  flex: 1 1 auto;
+}
+
+#X-popup {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+}
+
+#popupcontainer {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border: 1px solid #fff;
+  background: rgba(255, 255, 255, 0.16);
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  margin-left: 1.6rem;
+  margin-top: 2rem;
+  min-width: 0;
+  min-height: 0;
+  width: 90vw;
+  height: 100vh;
+  align-self: center;
+  overflow: hidden;
+}
```

Comment:
- [ ] Well done styling the popup window so far. However, you may have to write some more CSS code to shape the widths and heights of it across multiple screen sizes. Please take a look at the two images down below.
**Desktop**
![Screenshot (2)](https://user-images.githubusercontent.com/47625407/188758083-628aaa84-67de-42b8-ac8f-e76ab829622c.png)
**Mobile**
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/188758088-38087247-c2e3-422c-aa7f-213499d1b745.png)

- [ ] Kindly adjust your design to make it fit the multiple sizes of a screen 🙏